cookbook/reasoning/models/openai/o3_mini.py

```diff
@@ -1,7 +1,12 @@
 from agno.agent import Agent
 from agno.models.openai import OpenAIChat
 
-agent = Agent(model=OpenAIChat(id="o3-mini"))
+agent = Agent(
+    model=OpenAIChat(id="o3-mini"),
+    reasoning=True,
```

Comment:
The point of this example is to just have native reasoning. I agree it doesn't work, but then we should remove this example and add what we are trying to do here under the reasoning cookbooks.